components/Alias.tsx

```diff
@@ -14,6 +15,8 @@ const Alias = ({
   disabled?: boolean;
 }) => {
   const state = useContext(AppStateContext)!;
+  const tzktAlias = useTzktAccountAlias(address);
+  console.log("🚀 ~ tzktAlias:", tzktAlias);
```

Comment:
I don't know if logs has to be removed or not, if you think we should keep it we can